src/migrations/20221810114644-add-suggest-changes-table.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+CREATE TABLE IF NOT EXISTS suggest_change_set (
+  id serial primary key,
+  environment varchar(255) NOT NULL,
+  state varchar(255) NOT NULL,
+  project varchar(255) NOT NULL,
+  created_by integer not null references users (id) ON DELETE CASCADE,
+  created_at timestamp default now(),
+  updated_by integer not null references users (id) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS suggest_change (
+    id serial primary key,
+    action varchar(255) NOT NULL,
+    payload json NOT NULL,
+    created_by integer not null references users (id) ON DELETE CASCADE,
+    created_at timestamp default now(),
+    suggest_change_set_id integer NOT NULL REFERENCES suggest_change_set(id) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS suggest_change_event (
+    id serial primary key,
+    event varchar(255) NOT NULL,
+    data json NOT NULL,
```

Comment:
Same thing, please use jsonb unless you have really good reason for not doing so.

Reply:
Good idea